scripts/great-renaming.js

```diff
@@ -52,7 +52,7 @@ const rename = {
   storybook: 'storybook',
   tasks: 'task',
   templates: 'template',
-  trash: 'trash',
+  archive: 'archived',
```

Comment:
This didn't exist before, so doesn't need to be renamed. You could convert `trash: 'archived'`

Reply:
lol what is the storybook thing.